settingengine.go

```diff
@@ -236,6 +237,10 @@ func (e *SettingEngine) SetIncludeLoopbackCandidate(include bool) {
 	e.candidates.IncludeLoopbackCandidate = include
 }
 
+func (e *SettingEngine) SetUseTCPAllocationsForLocalRelayCandidates(use bool) {
```

Comment:
This is what we would set in goutils. Presumably in addition to having both the classic UDP `turn:turn.viam.com` and (re-)adding the all TCP `turn:turn.viam.com?transport=tcp`.